app.ts

```diff
@@ -1,22 +1,19 @@
+#!/usr/bin/env node
```

Comment:
Is this really needed?

Reply:
Yes, the shebang is needed. Otherwise, the following errors occur:
```                      
line 1: use strict: command not found
line 2: syntax error near unexpected token `('
line 2: `var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {'
```